Patches/Gas Traps and Shells/ThingDef_Misc/AmmoCategories_Gas.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Patch>
+
+	<Operation Class="PatchOperationFindMod">
+    <mods>
+		<li>Gas Traps And Shells</li>

```

Comment:
What's this about?

Reply:
No idea, it shouldn't be there as far as I'm aware.